src/main/java/Pet.java

```diff
@@ -0,0 +1,47 @@
+public class Pet {
+    private String name;
+    private int age;
+    private String species;
+    private Person owner;
+
+    public Pet(String name, int age, String species) {
+        this.name = name;
+        this.age = age;
+        this.species = species;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public void setAge(int age) {
+        this.age = age;
+    }
+
+    public String getSpecies() {
+        return species;
+    }
+
+    public void setSpecies(String species) {
+        this.species = species;
+    }
+
+    public Person getOwner() {
+        return owner;
+    }
+
+    public void setOwner(Person owner) {
```

Comment:
een ander optie is: deze kan korter door public void setOwner(Person owner){
this.owner = owner;
}
bij addPet in Person classe heb ik deze samengevoegd
  public void addPet(Person person, Pet pet) {
        List<Pet> pets = new ArrayList<>();
        if (person.getPets() != null) {
            for (Pet owner : getPets()) {
                pets.add(owner);
            }
        }
        pets.add(pet);
        person.setPets(pets);
    }